cfgov/v1/atomic_elements/organisms.py

```diff
@@ -560,3 +561,26 @@ class VideoPlayer(blocks.StructBlock):
     class Meta:
         icon = 'media'
         template = '_includes/organisms/video-player.html'
+
+
+class HTMLBlock(blocks.StructBlock):
+    html_url = blocks.RegexBlock(
+        label='Source URL',
+        default='',
+        required=True,
+        regex=r'^https://.+$',
+        error_messages={
+            'required': 'The HTML URL field is required for rendering raw HTML from a remote source.',
+            'invalid': 'The URL is invalid or not allowed. ',
+        }
+    )
+
+    def render(self, value, context=None):
+        resp = requests.get(value['html_url'])
```

Comment:
Don't forget to add a timeout per: http://docs.python-requests.org/en/master/user/quickstart/#timeouts